examples/doc-sync/fuludocs/README.md

```diff
@@ -0,0 +1,28 @@
+# Gallery in react
```

Comment:
Rename Gallery

Reply:
my bad! my last commit somehow was not pushed